### PR TITLE
Implement Submission Sheet Endpoint

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Add endpoint 
Wire to cso

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [X] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [X] No

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/filingpackage/filingpackageApiDelegateImpl/GetSubmissionSheetTest.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
+package ca.bc.gov.open.jag.efilingapi.filingpackage.filingpackageApiDelegateImpl;
+
+import ca.bc.gov.open.jag.efilingapi.Keys;
+import ca.bc.gov.open.jag.efilingapi.api.model.FilingPackage;
```
Comment (KDNeufeld) [inline src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/filingpackage/filingpackageApiDelegateImpl/GetSubmissionSheetTest.java:4]: unused import

File: src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/filingpackage/filingpackageApiDelegateImpl/GetSubmissionSheetTest.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    FilingPackageService filingPackageService;
+
+    @Mock
+    private SecurityContext securityContextMock;
+
+    @Mock
+    private Authentication authenticationMock;
+
+    @Mock
+    private KeycloakPrincipal keycloakPrincipalMock;
```
Comment (KDNeufeld) [inline src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/filingpackage/filingpackageApiDelegateImpl/GetSubmissionSheetTest.java:48]: Raw type.  Should this be
KeycloakPrincipal<KeycloakSecurityContext> ...?

File: src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/filingpackage/filingpackageApiDelegateImpl/GetSubmissionSheetTest.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+    }
+
+    @Test
+    @DisplayName("403: when no universal id should return 403")
+    public void withNoUniversalIdShouldReturn403() {
+
+        Mockito.when(tokenMock.getOtherClaims()).thenReturn(null);
+
+        ResponseEntity actual = sut.getSubmissionSheet(BigDecimal.ONE);
```
Comment (KDNeufeld) [inline src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/filingpackage/filingpackageApiDelegateImpl/GetSubmissionSheetTest.java:96]: Raw type.

**Comment by KDNeufeld** [comment]
Great work.  I can't explain the -0.2% code coverage.

File: src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/filingpackage/filingpackageApiDelegateImpl/GetSubmissionSheetTest.java (hunk 1 above)
Comment (TayGov) [reply src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/filingpackage/filingpackageApiDelegateImpl/GetSubmissionSheetTest.java]: 👍 I should enable the auto import manager

**Review by karthikmsqa — APPROVED** [review]
LGTM. Thanks!!

**Pull request merged**